src/utils/getPermissions.ts

```diff
@@ -24,26 +25,27 @@ const blockedPermission = () => {
   );
 };
 
-const handleGalleryPermissions = async (): Promise<PermissionStatus> => {
+const getPermissions = async (type: string): Promise<PermissionStatus> => {
   let result: PermissionStatus;
 
-  Platform.OS === 'android'
-    ? (result = await check(PERMISSIONS.ANDROID.READ_EXTERNAL_STORAGE))
-    : (result = await check(PERMISSIONS.IOS.PHOTO_LIBRARY));
+  result =
```

Comment:
use `const` instead of let in this case.

if type is necessary just use cast on it

Reply:
Done